frontend/src/components/InfoPanel.jsx

```diff
@@ -124,9 +126,9 @@ class InfoPanel extends React.PureComponent {
 
   async reloadStateData() {
     await Promise.all([
-      this.loadRestrictionData(),
+      await this.loadRestrictionData(),
+      await this.loadAllowedDates(),
```

Comment:
I'm not sure this is quite what we want. If we do:

```
await Promise.all([a, b, c]);
```

that will start `a`, `b` and `c` then wait until all three are finished. If ordering is important then I think we want:

```
await a;
await b;
await c;
```
which will return them one after the other instead of in parallel. In other words, if we want each one to finish before starting the next one, we can/should drop the `Promise.all` completely.

Reply:
Ok now I understand, thanks the explanation :). This is now implemented in 9436143.